components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -890,9 +872,8 @@ SUBROUTINE p3_main_main_loop(kts, kte, kbot, ktop, kdir, log_predictNc, dt, odt,
       nevapr(k)  = qisub + qrevp
       prer_evap(k) = qrevp
       vap_ice_exchange(k) = qidep - qisub + qinuc
-      vap_liq_exchange(k) = -qrevp + qcnuc
+      vap_liq_exchange(k) = - qrevp
       liq_ice_exchange(k) = qcheti + qrheti - qimlt + qiberg + qccol + qrcol
-      vap_cld_exchange(k) = qcnuc
```

Comment:
It looks like this variable is no longer needed, we may want to either delete it entirely from p3_main and from p3_interface.  Or just set it to zero here.  Currently it is unitialized but still written as output which could lead some in the future to believe there is vap_cld exchange and that it has weird values.  @hassanbeydoun  may have a better opinion on this since he is using this output in his own work.

Reply:
Urgh - this variable is truly going away because all liquid evaporation should be handled by SHOC (anything else is double counting). Thus I think we need to delete this output and this line.

Reply:
OK -- will delete `vap_cld_exchange` completely.